cypress/features/regression/fileInput.feature

```diff
@@ -0,0 +1,122 @@
+Feature: File Input component
+  I want to check File Input component properties
+
+  @positive
+  Scenario Outline: Attach <file> in File Input
+    Given I open "File Input" component page "default"
+    When I attach the file "<file>" to the "first" input
+    Then "<file>" file should be attached to the File Input
+      And "Remove" button is visible
+    Examples:
+      | file     |
+      | file.png |
+      | file.mov |
+
+  @positive
+  Scenario: Attach file with very long title in File Input - the file name should be formatted
+    Given I open "File Input" component page "default"
+    When I attach the file "file_with_very_long_title.png" to the "first" input
+    Then "file_with_...itle.png" file should be attached to the File Input
+      And "Remove" button is visible
+
+  @positive
+  Scenario: Delete attached file
+    Given I open "File Input" component page "default"
+      And I attach the file "file.png" to the "first" input
+      And "file.png" file should be attached to the File Input
```

Comment:
I'd prefer this condition to be more positive, i.e. "And "file.png" file is attached to the File Input"